src/sgame/CombatFeedback.cpp

```diff
@@ -77,15 +77,25 @@ void HitNotify(gentity_t *attacker, gentity_t *victim,
 	event->s.angles2[0] = damage;
 	event->s.otherEntityNum = victim - g_entities;
 
-	Vec3 point;
-	if (!point_opt || indirect)
-		point = Vec3::Load(victim->r.currentOrigin) +
-		        (Vec3::Load(victim->r.mins) +
-		         Vec3::Load(victim->r.maxs)) * 0.5f;
+	vec3_t point;
+	if ( !point_opt || indirect )
+	{
+		VectorCopy( victim->r.currentOrigin, point );
+		// those asserts compare floats, but said floats are
+		// read from config files. Original code just did the
+		// calculation for each dimensions, but I strongly doubt
+		// the engine supports assymetric BBoxes for horizontal
+		// plane. Putting asserts to make the assumption explicit.
+		ASSERT( fabs( victim->r.mins[ 0 ] ) == fabs( victim->r.maxs[ 0 ] ) );
```

Comment:
mins == -maxs would be a nicer way to write the assertion

Reply:
Seriously, I think it would be better to just have a single value instead of those 4 ones (X is always same as Y, mins is always -maxs...) this would make a lot of things easier, and use the memory space for more useful stuff like squared radius and radius...
But yes, what you propose seems a bit more readable.

Reply:
fixed